src/vm/sampleprofiler.cpp

```diff
@@ -195,4 +221,30 @@ void SampleProfiler::WalkManagedThreads()
     }
 }
 
-#endif // FEATURE_PERFTRACING
+void SampleProfiler::PlatformSleep(long nanoseconds){
+#ifdef FEATURE_PAL
+    PAL_nanosleep(nanoseconds);
+#else //FEATURE_PAL
+    ClrSleepEx(s_samplingRateInNs / 1 MILLION, FALSE);
+#endif //FEATURE_PAL
+}
+
+void SampleProfiler::BeginTimePeriod(){
+#ifndef PLATFORM_UNIX
```

Comment:
The calls to timeBeginPeriod only exist on Windows, so they should be in #ifdef PLATFORM_WINDOWS instead of #ifndef PLATFORM_UNIX.  It's essentially the same right now, but declares intentions more clearly.